x/distribution/migrations/v4/migrate_funds.go

```diff
@@ -9,23 +9,24 @@ import (
 )
 
 // MigrateFunds migrates the distribution module funds to pool module
-func MigrateFunds(ctx sdk.Context, bankKeeper types.BankKeeper, feePool types.FeePool, macc, poolMacc sdk.ModuleAccountI) error {
-	poolBal := sdk.NormalizeCoins(feePool.CommunityPool)
```

Comment:
Here the dust would stay in the community pool and as the previous migration was resetting it it would not match the supply